src/apps/content-editor/src/app/ContentEditor.js

```diff
@@ -37,57 +40,80 @@ export default function ContentEditor() {
   }, []);
 
   return (
-    <WithLoader
-      condition={navContent.nav.length || navContent.headless.length}
-      message="Starting Content Editor"
-    >
-      <section
-        className={cx(
-          styles.ContentEditor,
-          ui.contentNav ? styles.ContentGridOpen : "",
-          ui.contentNavHover && !ui.contentNav ? styles.ContentGridHover : ""
-          // true ? styles.ContentNavHover : ""
-        )}
-      >
-        <ContentNav
-          data-cy="contentNav"
-          dispatch={dispatch}
-          models={contentModels}
-          nav={navContent}
-        />
-
-        <ContentNavToggle />
-        <div
-          className={cx(
-            styles.Content,
-            ui.openNav ? styles.GlobalOpen : styles.GlobalClosed
-          )}
-        >
-          <div className={styles.ContentWrap}>
-            <Switch>
-              <Route exact path="/content" component={Dashboard} />
-              <Route exact path="/content/link/new" component={LinkCreate} />
-              <Route
-                exact
-                path="/content/:modelZUID/new"
-                component={ItemCreate}
-              />
-              <Route path="/content/link/:linkZUID" component={LinkEdit} />
-              <Route
-                exact
-                path="/content/:modelZUID/import"
-                component={CSVImport}
-              />
-              <Route
-                path="/content/:modelZUID/:itemZUID"
-                component={ItemEdit}
-              />
-              <Route exact path="/content/:modelZUID" component={ItemList} />
-              <Route path="*" component={NotFound} />
-            </Switch>
-          </div>
+    <Fragment>
+      {navContent.headless.length === 0 ? (
```

Comment:
We need to look `raw` data, which is the non segmented data from the api, to determine whether there are no models on the instance. `headless` only accounts for datasets and so an instance which contain single or multi page but no headless models would fail to render.